src/typechecker.erl

```diff
@@ -2232,8 +2232,10 @@ do_type_check_expr(Env, {'try', _, Block, CaseCs, CatchCs, AfterBlock}) ->
 
 %% Maybe - value-based error handling expression
 %% See https://www.erlang.org/eeps/eep-0049
-do_type_check_expr(_Env, {'maybe', Anno, [{maybe_match, _, _LHS, _RHS}]} = MaybeExpr) ->
-    erlang:throw({unsupported_expression, Anno, MaybeExpr}).
+do_type_check_expr(_Env, {'maybe', _, _}) ->
+    erlang:throw({skip, maybe_expr_not_supported});
```

Comment:
I suppose you are going to rebase this on top of the changes in #570, right?

Reply:
Yes, exactly

Reply:
I applied your suggestion here:

> maybe even a bit better solution would be to skip just the maybe expressions (instead of skipping whole functions) and assign the expression the type any().

It might enable working with a bit more real-world code, indeed.

Reply:
Thanks!